README.adoc

```diff
@@ -79,8 +79,7 @@ is in the `start/src` directory:
 
 == Testing Gradle
 
-If you do not have Gradle installed, running the Gradle Wrapper will automatically install Gradle. To learn 
-more about Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle documentation^].
+If you do not have Gradle installed, make sure that the `JAVA_HOME` environment variable is set, or that the java application can run. Running the Gradle Wrapper will automatically install Gradle. To learn more about Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle Wrapper documentation^].
```

Comment:
Suggested update:
"If you do not have Gradle installed, make sure that the `JAVA_HOME` environment variable is set, or that the **Java** application can run. Running the Gradle Wrapper **automatically installs** Gradle. To learn more about **the** Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle Wrapper documentation^]."

Reply:
Thanks!